README.md

```diff
@@ -0,0 +1,27 @@
+# OneBusAway skill for Alexa
+
+Are you ready to ask about your bus with just your voice?
+
+This project is an implementation of the [OneBusAway](http://onebusaway.org/) _open-source platform for real-time transit info_ for Amazon's [Alexa Voice Service](https://developer.amazon.com/public/solutions/alexa/alexa-voice-service).
+
+## Install and Use
+
+This project isn't released publicly yet.  We plan to make the first release around January 2016.
+
+Initial spoken setup: TBD
+
+Supported voice commands: TBD
+
+## Develop
+
+1. Install the [Java Platform SE Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and [Maven](https://maven.apache.org/).
+2. Clone `https://github.com/amzn/alexa-skills-kit-java`, preferably to `${user.home}/Repos/alexa-skills-kit-java".

```

Comment:
This is unfortunately (and unnecessarily) messy, due to https://github.com/amzn/alexa-skills-kit-java/issues/1.  Instead of having to clone the ASK project, I'd prefer to just bundle the ASK JAR in the Git repo and refer to it as a local dependency in Maven.  When Amazon fixes this we can switch to pulling the ASK dependency from Maven Central.